### PR TITLE
Info about VS community and IIS

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md [hunk 1]
```diff
@@ -401,3 +401,13 @@ To connect to the docker on SSL, both the docker image and your local machine mu
 In the future, passing in a custom cert via a bound volume that contains your certificate will be a possibility as well.
 
 For additional reading on this process for trusting SSL certs locally, feel free to read up [here.](https://devblogs.microsoft.com/aspnet/configuring-https-in-asp-net-core-across-different-platforms/) The afore-mentioned [docker specific readme](../docker/README.md) also has details that are relevant.
+
+## Troubleshooting
+
+### Visual Studio - Community edition
+
+There's an issue with the Visual Studio Community edition. Use Enterprise to run locally.
```
Comment (benbp) [inline tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:409]: It would be helpful to explain what the issue is in case it is fixed in the future.

Comment (vhvb1989) [reply tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:409]: I don't know what it is =D. It just throw an error about the app can start.

Comment (vhvb1989) [reply tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:409]: image

Comment (vhvb1989) [reply tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:409]: I just found out the issue 😄   updating pR in a moment

**Comment by vhvb1989** [comment]
new PR for this: https://github.com/Azure/azure-sdk-tools/pull/2665

**Pull request closed without merging**